src/components/Form/Form.js

```diff
@@ -37,12 +39,13 @@ export default function Form({ id }) {
 		}
 	}, [entryToUpdate, setValue]);
 
-	const onSubmit = (data, event) => {
+	const onSubmit = async (data, event) => {
 		if (entryToUpdate) {
 			controlEntry(id, data);
 			editEntry(id);
 		} else {
-			addEntry(data);
+			const geoData = await fetchAddressData(watch('address'));
+			addEntry({ ...data, position: [Number(geoData[0].lat), Number(geoData[0].lon)] });
 			event.target.reset();
 			modalShow();
 		}
```

Comment:
Maybe you want your user see the made entry after submit? Actually, the user stay on the create-page after submitting the form. You can use router to push the user either to the map or to the list page.

Reply:
I have already mentioned this in one of my next user stories, thanks! :)